arrow-site/docs/_data/sidebar-fx.yml

```
@@ -27,15 +23,28 @@ options:
       - title: CircuitBreaker
         url: /apidocs/arrow-fx-coroutines/arrow.fx.coroutines/-circuit-breaker/
 
+
+  - title: Additional information
```

```suggestion
  - title: Misc
```
Trying to make it shorter but the current one looks good also